blackbox/docs/general/builtins/scalar.rst

```diff
@@ -2104,6 +2104,33 @@ Examples::
     +------------------------------+
     SELECT 1 row in set (... sec)
 
+
+``pg_get_expr``
+---------------
+
+.. NOTE::
+
+   The function is implemented to improve compatibility with clients that use
+   the PostgreSQL wire protocol. The function always returns ``null``.
+
+The ``pg_get_expr`` function decompiles internal form of an expression.
```

Comment:
this doesn't mean anything to me (and postgres documents this quite cryptically imo) I think we should either document it in a more ... human understandable  way or drop it completely (I'd say we drop it completely as it just returns null)